src/rules/spaceWithinParensRule.ts

```diff
@@ -116,10 +120,10 @@ class SpaceWithinParensWalker extends Lint.AbstractWalker<Options> {
             currentChar = this.sourceFile.text.charCodeAt(currentPos);
         }
         /**
-         * Number 40 is open parenthese char code, we skip this cause
-         * it's already been caught by `checkOpenParenToken`
+         * Skip the open parenthesis character because it
+         * has already been caught by `checkOpenParenToken`
          */
-        if (!ts.isLineBreak(currentChar) && currentChar !== 40) {
+        if (!ts.isLineBreak(currentChar) && this.sourceFile.text.charAt(currentPos) !== ts.tokenToString(ts.SyntaxKind.OpenParenToken)) {
```

Comment:
I don't see a reason to replace the char code. If you don't like the magic number, you can add a `const enum` (because they are inlined by the compiler) with the charcode.

Reply:
Thanks for the feedback, I put back the previous code.